app/models/user.rb

```diff
@@ -45,13 +46,33 @@ def remember
     update_column(:remember_digest, User.digest(remember_token))
   end
 
-  def authenticated? remember_token
-    return false unless remember_digest
+  def authenticated? attribute, token
+    digest = send "#{attribute}_digest"
+    return false if digest.nil?
```

Comment:
Code cũ vẫn ổn mà em
```suggestion
    return false unless digest
```